.pyproject_generation/pyproject_custom.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "hexkit"
-version = "4.0.0"
+version = "4.1.0"
```

Comment:
You do not need to bump the version since 4.0.0 has not yet been released.

Reply:
Addressed in 0fa7eb3935636c9349c1609fbba698d1558b5417